src/components/views/auth/ModularServerConfig.js

```diff
@@ -77,19 +77,22 @@ export default class ModularServerConfig extends React.PureComponent {
         });
     }
 
-    onHomeserverChanged = (ev) => {
-        this.setState({hsUrl: ev.target.value}, () => {
-            this._hsTimeoutId = this._waitThenInvoke(this._hsTimeoutId, () => {
-                let hsUrl = this.state.hsUrl.trim().replace(/\/$/, "");
-                if (hsUrl === "") hsUrl = this.props.defaultHsUrl;
-                this.props.onServerConfigChange({
-                    hsUrl: this.state.hsUrl,
-                    isUrl: this.props.defaultIsUrl,
-                });
+    onHomeserverBlur = (ev) => {
+        this._hsTimeoutId = this._waitThenInvoke(this._hsTimeoutId, () => {
+            let hsUrl = this.state.hsUrl.trim().replace(/\/$/, "");
+            if (hsUrl === "") hsUrl = this.props.defaultHsUrl;
+            this.props.onServerConfigChange({
+                hsUrl: this.state.hsUrl,
```

Comment:
should this be `hsUrl: hsUrl`? seems like the local variab;e hsUrl is never read? Maybe I'm nuts, as the linter isn't complaining, but I'm not seeing it.

Reply:
Ah interesting, I guess this was a bug in the existing code...? The linter thinks it's okay because `if (hsUrl === "") hsUrl = this.props.defaultHsUrl;` qualifies as reading the variable.

Thanks for catching, I'll update!

Reply:
Hmm, I think I'll actually strip out this unused code for now. (I [confirmed](https://github.com/matrix-org/matrix-react-sdk/blob/17e9a442e447db620908a1be908449c365a52218/src/components/views/login/ServerConfig.js#L93) it was also dead code in release as well.)